### ISSUE TITLE
logged user name

### ISSUE THREAD

**Issue by apps4u** [OraOpenSource/Logger#236]
Hi Im not sure this is a bug I have logger install on two servers one for dev the other production. But for some reason I get different users names when logging. 1 always has the user a APEX_PUBLIC_USER as the user name (production 19.1) while dev server shows the user_name as the LDAP user who's logged into apex.

I just created a quick test and I called logger but got different user when the same use was logged into both.
<img width="1880" alt="Screen Shot 2020-05-07 at 3 13 01 pm" src="https://user-images.githubusercontent.com/756522/81256978-6814fa00-9075-11ea-83ee-e96309b798fa.png">
<img width="1877" alt="Screen Shot 2020-05-07 at 3 12 56 pm" src="https://user-images.githubusercontent.com/756522/81256980-6b0fea80-9075-11ea-96d5-48144641899b.png">

Is this a bug Im not sure why they would be different the Oracle Apex workspace is the same on both hosts I copied from dev to production when we created the new workspace so they both have the same work_space_id and both have the same setting and apex version the only difference is that Oracle Database  is 12.2c and the other 19.1c both are on apex 20.1

**Comment by apps4u** [comment]
As I said Im not sure if its me just making a mistake or a bug as I have the same settings on both hosts the only difference is the database version not Apex and all Logger version are the same

APEX 20.1
ORDS 19.2
Oracle DB, 12.2c & 19.1c
logger version 3.1.1

**Comment by martindsouza** [comment]
@apps4u I don't think this is a Logger problem. When you see the `APEX_PUBIC_USER` being logged what is the `APP_USER` in APEX showing?
